scripts/build/build_node/Platform/Linux/package-list.ubuntu-bionic.txt

```diff
@@ -1,9 +1,9 @@
 # Package list for Ubuntu 18.04
 
 # Build Tools Packages
-cmake/3.20.1-0kitware1ubuntu18.04.1     # For cmake
-clang-12                                # For the compiler and its dependencies
-ninja-build                             # For Ninja Build System
+cmake/3.24.0-0kitware1ubuntu18.04.1     # For cmake
+clang-12                                # For Ninja Build System
+ninja-build                             # For the compiler and its dependencies
```

Comment:
These comments got swapped.

Reply:
I think it means that `clang-12` is used for the Ninja build system, but I can make that more clear

Reply:
Sure, fixing it